app/src/main/java/org/apache/taverna/mobile/ui/workflowrun/WorkflowRunActivity.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.taverna.mobile.ui.workflowrun;
+
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentActivity;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentStatePagerAdapter;
+import android.support.v4.view.PagerAdapter;
+import android.widget.Toast;
+
+import com.anton46.stepsview.StepsView;
+
+import org.apache.taverna.mobile.R;
+import org.apache.taverna.mobile.data.DataManager;
+import org.apache.taverna.mobile.data.local.PreferencesHelper;
+import org.apache.taverna.mobile.ui.DownloadingFragment;
+import org.apache.taverna.mobile.ui.playerlogin.PlayerLoginFragment;
+import org.apache.taverna.mobile.utils.NonSwipeableViewPager;
+import org.apache.taverna.mobile.utils.WebViewGen;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+import static com.raizlabs.android.dbflow.config.FlowManager.getContext;
+
+public class WorkflowRunActivity extends FragmentActivity implements WorkflowRunMvpView,
+        PlayerLoginFragment.OnSuccessful {
+
+    public static final String WORKFLOW_URL = "Workflow_url";
+    private final String[] labels = {"Player\nLogin", "Download", "Upload", "Inputs &\n Run"};
+    @BindView(R.id.stepsView)
+    StepsView mStepsView;
+    @BindView(R.id.viewpager)
+    NonSwipeableViewPager mPager;
```

Comment:
give a single line space between the variables, As you have managed in other fragments.